src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -186,10 +234,26 @@ void HwndTerminal::SendOutput(std::wstring_view data)
 
 HRESULT _stdcall CreateTerminal(HWND parentHwnd, _Out_ void** hwnd, _Out_ void** terminal)
 {
-    auto _terminal = std::make_unique<HwndTerminal>(parentHwnd);
+    auto _hostWindow = CreateWindowEx(
+        0,
+        L"static",
+        nullptr,
+        WS_CHILD |
+            WS_CLIPCHILDREN |
+            WS_CLIPSIBLINGS |
+            WS_VISIBLE,
+        0,
+        0,
+        0,
+        0,
+        parentHwnd,
+        (HMENU)0x02,
+        nullptr,
+        0);

```

Comment:
Comment as to why this additional HWND is being created?

Reply:
I'll add a comment. The reason is that in order for HwndHosts to properly plumb together the UIA tree you cannot host a control directly inside of an `HwndHost`. The control _must_ be hosted inside an actual top level window, which in this case is a window with the `"static"` class. This took a couple months of on/off investigation.